classes/PodsAPI.php

```diff
@@ -3792,16 +3792,18 @@ public function save_relationships ( $id, $related_ids, $pod, $field ) {
                 $object_type = 'post';
 
             delete_metadata( $object_type, $id, $field[ 'name' ] );
-
-            if ( !empty( $related_ids ) ) {
-                update_metadata( $object_type, $id, '_pods_' . $field[ 'name' ], $related_ids );
-
-                foreach ( $related_ids as $related_id ) {
-                    add_metadata( $object_type, $id, $field[ 'name' ], $related_id );
-                }
+            delete_metadata( $object_type, $id, '_pods_' . $field[ 'field' ] );
+            
+            if ( 1 < count( $related_ids ) ) {
```

Comment:
Wrap this `if` only around the `update_metadata`, inside the `if ( ! empty( $related_ids ) ) {`